Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailCellBuilder.swift

```diff
@@ -431,10 +414,17 @@ class TransactionDetailCellBuilder {
                 tx.detailedExecutionInfo else {
             return
         }
+
+        var isRejection = false
+
+        if case let SCGModels.TxInfo.rejection(_) = tx.txInfo {
```

Comment:
Is this thing needed anywhere else? Maybe make a file private extension on `TxInfo` in this file and use it here :wink:

Reply:
Good idea, I made it not as file private because I think we can benefit from more in feature